modules/base/acl/module.py

```diff
@@ -471,16 +475,16 @@ async def acl_overwrite_role_remove(self, ctx, command: str, role: discord.Role)
 
     @check.acl2(check.ACLevel.SUBMOD)
     @acl_overwrite_role_.command(name="list")
-    async def acl_overwrite_role_list(self, ctx):
+    async def acl_overwrite_role_list(self, ctx: commands.Context):
         """List ACL role overwrites."""
         ros = RoleOverwrite.get_all(ctx.guild.id)
 
         class Item:
-            def __init__(self, obj):
-                role = ctx.guild.get_role(obj.role_id)
-                self.role = getattr(role, "name", str(obj.role_id))
-                self.command = obj.command
-                self.allow = _(ctx, "yes") if obj.allow else _(ctx, "no")
+            def __init__(self, ro: RoleOverwrite):
+                role = ctx.guild.get_role(ro.role_id)
+                self.role = getattr(role, "name", str(ro.role_id))
+                self.command = ro.command
```

Comment:
This has to be addressed in the future - the short names such as `ro` should be extended to `role_overwrite` for clarity. Let's keep it this way for now.